draft-ietf-tls-rfc8446bis.md

```diff
@@ -4102,6 +4103,13 @@ decrypt_error:
   to correctly verify a signature or validate a Finished message
   or a PSK binder.
 
+ticket_invalid:
```

Comment:
"ticket" has a very specific connotation that I think you might want to avoid here.  How about `psk_rejected`?  My only reservation there is that it looks like your goal is to signal that the psk *identifier* is not acceptable to the server.